lib/rules/prop-types.js

```diff
@@ -301,6 +301,17 @@ module.exports = {
     }
 
     /**
+     * Removes quotes from around an identifier.
+     * @param {string} the identifier to strip
+     */
+    function stripQuotes(string) {
+      if (string[0] === '\'' || string[0] === '"') {
```

Comment:
[Flow doesn't seem to like them](https://flowtype.org/try/#0C4TwDgpgBACgTgezAZygXigbwFBSgAwGMEBbMAGwmAgH58AuKAOwFcSAjCOAGmwF8A3EA)